tests/conftest.py

```diff
@@ -885,7 +885,7 @@ def _generate_random_skeleton(**extra_kwargs):
         "allow_overlap": True,
     }
     # kwargs.update
-    heights = {"scale": 100, "sigma": 5.0, "cval": 20.0}
+    heights = {"scale": 1e2, "sigma": 5.0, "cval": 20.0}
```

Comment:
Are these not the same value?

Reply:
Sorry for the confusion I caused, should have posted here after pushing. But aye, what Sylvia, said it was super weird as although the values `100` and `1e2` are the same, they have a very different effect when used in scale:
![Screenshot 2024-10-14 at 17 28 51](https://github.com/user-attachments/assets/55ebff94-b8df-422b-b29f-cb0b831ed734)

And the `1e2` is far from very small values which seem to give floating point errors.